service/src/main/resources/application.yml

```diff
@@ -173,6 +174,7 @@ workspace:
     protected-data-landing-zone-defs: ["ProtectedDataResourcesFactory"]
     azure-database-util-image: "us.gcr.io/broad-dsp-gcr-public/azure-database-utils:${version.gitHash}"
     azure-database-util-logs-tail-lines: 1000
+    azure-environment: ${env.azure.environment}
```

Comment:
Is this setting required? It seems to be a duplicate from the one above.

Reply:
The first one pulls it from the OS environment, the second one is where the Workspace Azure config object reads it from and refers to the first one. Seems to be the pattern used. @dvoet Am I doing this config setup correctly?